core/server/master/src/main/java/alluxio/master/metrics/DefaultMetricsMaster.java

```diff
@@ -53,6 +57,22 @@
   private final Set<MultiValueMetricsAggregator> mMultiValueMetricsAggregatorRegistry =
       new HashSet<>();
   private final MetricsStore mMetricsStore;
+  private final HeartbeatThread mClusterMetricsUpdater;
+
+  /**
+   * Heartbeat executor that updates the cluster metrics.
+   */
+  private class ClusterMetricsUpdater implements HeartbeatExecutor {
```

Comment:
I think we usually put inner classes towards the bottom of the file

Reply:
Updated, thanks.